### PR TITLE
[ET][000000] fixing security issue with ruby-saml, updated readme

### PR DESCRIPTION
**Pull request by Ebtoulson**

### CODE REVIEW

File: lib/okta_saml/session_helper.rb [hunk 1]
```diff
@@ -2,7 +2,11 @@ module OktaSaml
   module SessionHelper
     def sign_in(user)
       cookies.signed[:remember_token] = {
-        :value => user.email
+        value: {
```
Comment (armondhonore) [inline lib/okta_saml/session_helper.rb]: This looks like you added some features

**Comment by armondhonore** [comment]
@Ebtoulson this merge looks like it also adds some functionality not just a security fix

**Comment by Ebtoulson** [comment]
@armondhonore lol what you doing watching this repo.
Yea, its the same functionality from https://github.com/rentpath/okta_saml/pull/5 which actually had a bug https://github.com/rentpath/okta_saml/commit/614f7b0948e6c697107ee96c588a04ea364468e0. I just didnt mention it in the msg

**Comment by Ebtoulson** [comment]
I figured it would be nice to get it back to a state that anyone could use it before changing maintainers. The lack of tests is very un-nerving :worried: but everything seemed to work with the dummy app.

**Pull request merged**